docs/source/release/v6.1.0/framework.rst

```diff
@@ -14,16 +14,14 @@ Concepts
 
 Algorithms
 ----------
-:ref:`LoadNexusLogs <algm-LoadNexusLogs>` has additional parameters to allow or block specific logs from being loaded.
```

Comment:
What happened to these 2 notes?

Reply:
I see that resolving conflicts and merging the release notes went pretty poorly. I'll correct it.